samples/Kconfig

```diff
@@ -16,15 +16,54 @@ config NCS_SAMPLES_DEFAULTS
 	help
 	  Use the default configuration for NCS samples.
 
-config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE
+if SOC_SERIES_NRF53X
+
+config AUTO_INCLUDE_CHILD_IMAGE
+	bool "Automatically add a child image when relevant"
```

Comment:
no it works automatically because of the `imply` that is being set by the other configs like multiproto, hci_rpmsg etc, and they are in turn enabled by default when certain conditions are met (e.g. ble, ble & zigbee, etc). We'd like to keep it that way so users that just want to use BLE on an nRF53 device don't have to faff around looking for the right kconfig when building a zephyr sample.

Reply:
> agreed that I often forget to open menuconfig to check how it looks (since I don't use it that often), will try to remember to do that from now on.

Great.
And this is actually very important cause we describe to users of nRF Connect SDK that this is a recommended way to change configuration temporarily:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#changing-the-configuration-temporarily

Reply:
looks a bit better now:
![image](https://user-images.githubusercontent.com/68428270/137886241-917e2278-3431-4704-822d-1cbd673ab05e.png)